From 12b1316d61b7f49fb6a12b4e6b46383a3ea46c2e Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Fri, 19 Oct 2018 12:08:52 +0200
Subject: [PATCH 2/2] Merge android-9.0.0_r12

commit a8ce08e190ecc1c54fc4a4a115f4775e5b666c86
Merge: 6e0e4402 005ccd94
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Fri Jul 20 23:42:41 2018 +0000

    Snap for 4905103 from 005ccd94b9b137f804252c12234edac615d60d89 to pi-dr1-release

    Change-Id: I9fe2a0c1c0b7faf65f8c193cb48eb33c9de7c61d

commit 005ccd94b9b137f804252c12234edac615d60d89
Author: Bill Yi <byi@google.com>
Date:   Fri Jul 20 10:58:04 2018 -0700

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: If4ff0edaf7892899eacfab992791e23c65aa0909

commit 6e0e4402236ea910b6857e518e33a31318e80beb
Merge: 1481da79 40ac4a06
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Sun Jun 24 03:05:12 2018 +0000

    Snap for 4859140 from 40ac4a061c23f379aba4c3883f7e6a871ae01482 to pi-dr1-release

    Change-Id: I01a22c93e3218191aaac6a501c379254d7c2a57a

commit 40ac4a061c23f379aba4c3883f7e6a871ae01482
Author: Bill Yi <byi@google.com>
Date:   Thu Jun 21 07:03:22 2018 -0700

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: I9880a99d3b3b196da2d56440e3ace9f4573e45b9

commit 1481da792d43c5c840700b92425128c59b424110
Merge: e1cedfe5 d9bc5eb3
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Tue Jun 5 03:05:50 2018 +0000

    Snap for 4820872 from d9bc5eb37198434193d3ad5ff3b8d5110bd59a49 to pi-dr1-release

    Change-Id: I8bc2ace870d9d107bd4ccef53cf8d780a1414341

Change-Id: I042ee4ea3881158097c0a6a200d0c3d2cc86e6ce
---
 res/values-fr/strings.xml | 2 +-
 res/values-iw/strings.xml | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/res/values-fr/strings.xml b/res/values-fr/strings.xml
index 29bf594c..ebcaae14 100644
--- a/res/values-fr/strings.xml
+++ b/res/values-fr/strings.xml
@@ -74,7 +74,7 @@
     <string name="create_error" msgid="3092144450044861994">"Échec de la création du dossier"</string>
     <string name="query_error" msgid="6625421453613879336">"Impossible de charger le contenu pour le moment"</string>
     <string name="root_recent" msgid="1080156975424341623">"Récents"</string>
-    <string name="root_available_bytes" msgid="8269870862691408864">"Espace disponible : <xliff:g id="SIZE">%1$s</xliff:g>"</string>
+    <string name="root_available_bytes" msgid="8269870862691408864">"<xliff:g id="SIZE">%1$s</xliff:g> disponible(s)"</string>
     <string name="root_type_service" msgid="6521366147466512289">"Services de stockage"</string>
     <string name="root_type_shortcut" msgid="6059343175525442279">"Raccourcis"</string>
     <string name="root_type_device" msgid="1713604128005476585">"Appareils"</string>
diff --git a/res/values-iw/strings.xml b/res/values-iw/strings.xml
index eccd206a..ede64792 100644
--- a/res/values-iw/strings.xml
+++ b/res/values-iw/strings.xml
@@ -206,7 +206,7 @@
     <string name="open_external_dialog_root_request" msgid="6776729293982633">"האם להעניק לאפליקציה <xliff:g id="APPNAME"><b>^1</b></xliff:g> גישה לנתונים שלך, כולל תמונות וסרטונים, השמורים ב<xliff:g id="STORAGE"><i>^2</i></xliff:g>?"</string>
     <string name="never_ask_again" msgid="525908236522201138">"ראיתי פעם אחת, זה מספיק"</string>
     <string name="allow" msgid="1275746941353040309">"כן, זה בסדר"</string>
-    <string name="deny" msgid="5127201668078153379">"לא, אין מצב"</string>
+    <string name="deny" msgid="5127201668078153379">"לא, תודה"</string>
     <plurals name="elements_selected" formatted="false" msgid="4448165978637163692">
       <item quantity="two"><xliff:g id="COUNT_1">%1$d</xliff:g> נבחרו</item>
       <item quantity="many"><xliff:g id="COUNT_1">%1$d</xliff:g> נבחרו</item>
-- 
2.17.1

